lib/simple_forms_api_submission/metadata_validator.rb

```diff
@@ -36,9 +36,14 @@ def self.validate_file_number(metadata)
       metadata
     end
 
-    def self.validate_zip_code(metadata)
-      validate_presence_and_stringiness(metadata['zipCode'], 'zip code')
-      metadata['zipCode'] = '00000' unless metadata['zipCode'].match?(/\A\d{5}(-\d{4})?\z/)
+    def self.validate_zip_code(metadata, zip_code_is_us_based)
+      zip_code = metadata['zipCode'].dup
+      validate_presence_and_stringiness(zip_code, 'zip code')
+
```

Comment:
If I may make one suggestion, that any non-digits be deleted from the `zip_code` prior to performing any conversion. There was one instance where a metadata zip code value had the format `12345 6789`, which failed validation. That example zip code would be converted to `00000` in this method, instead of a what more accurately should be `12345-6789`.